backend/oidc/services.py

```diff
@@ -0,0 +1,13 @@
+from oidc.models import OIDCProfile
+
+
+def update_or_create_oidc_profile(user, defaults) -> OIDCProfile:
+    oidc_profile, _ = OIDCProfile.objects.update_or_create(
+        user=user,
+        defaults=defaults,
+    )
+    return oidc_profile
+
+
+def clear_oidc_profile(oidc_profile: OIDCProfile) -> None:
+    OIDCProfile.objects.filter(id=oidc_profile.id).delete()
```

Comment:
Is there any specific reason to not just use the `oidc_profile.delete()` here?

Reply:
Yep, I rewrote this and forgot to change that :+1: